tests/FSharp.Compiler.ComponentTests/Conformance/Types/TypeConstraints/IWSAMsAndSRTPs/IWSAMsAndSRTPsTests.fs

```diff
@@ -1145,7 +1145,7 @@ module StaticAbstractBug =
         |> compile
         |> shouldFail
         |> withDiagnostics [
-            (Error 1, Line 14, Col 41, Line 14, Col 42, "The type 'bool' does not match the type 'int'")
+            (Error 1, Line 14, Col 41, Line 14, Col 42, "The type 'int' does not match the type 'bool'")
             (Error 1, Line 16, Col 32, Line 16, Col 33, "This expression was expected to have type
```

Comment:
<img width="2815" height="484" alt="Image" src="https://github.com/user-attachments/assets/2a38c446-0a0b-4e85-93d4-080ed5a956a9" />

Here `expected = int, actual = bool`, which is incorrect. Just two lines below is a similar case, where `expected = bool` is correct.